File: 3154-maximum-value-of-an-ordered-triplet-i/maximum-value-of-an-ordered-triplet-i.c
long long maximumTripletValue(int* nums, int numsSize) {
    long long int max=0;
    for(int i=0;i<numsSize;i++){
        for(int j=i+1;j<numsSize;j++){
            for(int k=j+1;k<numsSize;k++){
                long long int y=nums[i]-nums[j];
                long long int res=0;
                if(y>0){
                    long long int x=y*nums[k];
                    res=x;
                }
                if(max<res){
                    max=res;
                }
            }
        }
    }
    return max;
    
}

File: 3154-maximum-value-of-an-ordered-triplet-i/maximum-value-of-an-ordered-triplet-i.py
class Solution:
    def maximumTripletValue(self, nums: List[int]) -> int:
        res=[]
        for i in range(0,len(nums)):
            for j in range(i+1,len(nums)):
                for k in range(j+1,len(nums)):
                    y=nums[i]-nums[j]
                    if y>0:
                        x=y*nums[k]
                        res.append(x)
        if len(res)==0:
            return 0
        else:
            return max(res)